server/endpoint/health/searcher/response.go

```diff
@@ -1,5 +1,33 @@
 package searcher
 
+type GeneralStatus struct {
+	Health       string `json:"health"`
+	Creating     bool   `json:"creating"`
+	Upgrading    bool   `json:"upgrading"`
+	Deleting     bool   `json:"deleting"`
+	Normal       bool   `json:"normal"`
+	NodeCount    int    `json:"nodeCount"`
+	MaxNodeCount int    `json:"maxNodeCount"`
+}
+
+type NodeStatus struct {
```

Comment:
There should be more structure here. E.g.
```
NodeStatus
    Limit
        CPU
```
And so on.
That produces more structs but makes it easier to change specific parts and extend the structure.

Reply:
That makes sense. I've cleaned this up now.